students/slawinski_amadeusz/lesson_03_functions/8.02_negative_exponent.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+
+def power(a, n):
+    result = 1
+    while n:
```

Comment:
Dlaczego używasz pętli `while` zamiast `for` w sytuacji gdy mamy znaną liczbę powtórzeń?